config/answers.sam-installer.yaml

```diff
@@ -3,7 +3,9 @@ certs:
   generate: true
   deploy: true
   group: foreman
-katello: true
+katello:
+  package_names:
+    - katello-sam
```

Comment:
@ehelms might be tricky to require rubygems-katello because of scl_prefix here...have to think of a way to denote that in package_names